src/components/ItemEditorPage/TopPanel/TopPanel.tsx

```diff
@@ -90,6 +94,19 @@ export default class TopPanel extends React.PureComponent<Props, State> {
     )
   }
 
+  // There will be an ItemCuration interface introduced later on a follow-up PR.
+  // For now let's use the current `Curation` interface for the itemCurations array.
+  renderTPButtons = (collection: Collection, collectionCuration: Curation | null, itemCurations: Curation[] | null) => {
+    const shouldShowApproveButton = itemCurations?.some(itemCuration => itemCuration.status === CurationStatus.PENDING)
+    return (
+      <>
+        {shouldShowApproveButton ? this.renderButton(ButtonType.APPROVE, collection, collectionCuration) : null}
```

Comment:
For now, approving the TP items will trigger the same approval flow as before, calling the  `onInitiateApprovalFlow`. That might change once the new sagas are available.